.dockerignore

```diff
@@ -0,0 +1,4 @@
+deployments
+docs*
+examples*
+grafana
```

Comment:
Should `tests`, `perf-tests` also be ignored?

Reply:
Adding a comment for the record, they can't be added because tests are used in docker